src/translate/en.js

```diff
@@ -68,5 +76,7 @@ export default {
   area: {
     Arktis: "Arctic",
     Antarktis: "Antarctic",
+    ["Generell art"]: "Generic",
```

Comment:
What does "Generell art" mean? Where does it come from?

If needed please make the label English.

Reply:
It was a feature asked by Oddveig: sometimes the cases are not located geographically and are "av generell art" (quoting Oddveig on the Norwegian term). The label is not meant to be English, as Artiks and Antarktis (those are the values returned by the API, and they are in Norwegian). The best equivalent in English that came to mind was 'generic' but I'll take anything better!

Reply:
About extra (non-)area.

The problem is really:

1. there is a bug changing/removing area from a case.
2. there is no obvious UI to filter on empty area

However, you can already filter on "null". 
First goto: 
https://placenames.npolar.no/?sort=-beginLifespanVersion&area=null

Then press "SAKER" and you should see 84 cases

Ref API
https://v2-api.npolar.no/placename/case/_search?and=area:null&show=name,id,@id
https://v2-api.npolar.no/placename/feature/_search?and=area:null&show=name,id

Reply:
So you recommend I just add the null option in the filtering and tell Oddveig that he can register a case with no area if he wants to be generic?

Reply:
Yes, I'd close this and create a new branch for fixing (setting, filtering) cases with no areas.
As well as handling/fixing these. No area is set, but names are connected (26):
https://v2-api.npolar.no/placename/case/_search?and=area:null&not=placenames.id:null&show=name,id,@id,placenames

Reply:
The 26 cases with no area AND name(s) are a bit weird, but if a "generic / no area" case is implemented or has consequences for only eg. Svalbard, this is allright, I guess. 

So by handling/fixing I meant really just indicating this situation for the editor, like
"Generic case, but connected to names in: Svalbard (51)":
 
https://stadnamn.npolar.no/sak/283